website/docs/core-concepts/data-access-and-binding/displaying-data-read/display-data-tables.md

```diff
@@ -1,98 +0,0 @@
-# Display Data in Tables
```

Comment:
Just adding it here so we don't forget to add a redirect from this page to the new section. Also in the Dusplay data page, you can cross-reference to this section so that people can find it from there